diff --git a/node_modules/node-stdlib-browser/esm/mock/empty.js b/node_modules/node-stdlib-browser/esm/mock/empty.js
index 4bf4979..50e1088 100644
--- a/node_modules/node-stdlib-browser/esm/mock/empty.js
+++ b/node_modules/node-stdlib-browser/esm/mock/empty.js
@@ -1,4 +1,6 @@
 var empty = null;
+export const promises = null;
+export const createReadStream = null;
 
 export { empty as default };
 //# sourceMappingURL=empty.js.map
